src/controllers/asteroid-web-controller.js

```diff
@@ -0,0 +1,17 @@
+const { mapQueryToMeteorRequest } = require('../utils');
+const { asteroidService } = require('../services');
+
+const displayAsteroids = async (req, res, next) => {
+  const request = mapQueryToMeteorRequest(req.query);
+
+  try {
+    const data = await asteroidService.getAsteroidsWithinPeriod(request);
+    res.render('meteors.html', { ...data, title: 'Asteroids' });
```

Comment:
What is the difference beween web and api controllers? Maybe you wanted api to return res.json(data)?